app/view/fund_transfer.py

```diff
@@ -0,0 +1,152 @@
+from app import db
+from app.Schema.account_transaction_details_schema import fund_transfer_schema, account_transaction_detail_schema
+from app.model.account_transaction_details import FundTransfer, TransactionType, AccountTransactionDetails
+from flask import request
+from flask_restful import Resource
+from app.common.ResponseGenerator import ResponseGenerator
+from flask_api import status
+from app.common.logging import *
+from app.model.bank_account import BankAccount
+
+
+class FundTransferInfo(Resource):
+
+    def post(self):
+
+        """Add fund transfer in the FundTransfer table"""
+        try:
+            fund_data = request.get_json()
+            result = fund_transfer_schema.validate(fund_data)
+            if result:
+                logger.exception(result)
+                response = ResponseGenerator(data={}, message=result, success=False,
+                                             status=status.HTTP_400_BAD_REQUEST)
+                return response.error_response()
+            fund = FundTransfer(from_account=fund_data['from_account'],
+                                to_account=fund_data['to_account'])
+            db.session.add(fund)
+            db.session.commit()
+            output = fund_transfer_schema.dump(fund)
+            from_account = BankAccount.query.filter(BankAccount.account_number == fund_data['from_account']).first()
+            to_account = BankAccount.query.filter(BankAccount.account_number == fund_data['to_account']).first()
+            transaction_type = TransactionType.query.filter(TransactionType.transaction_type == "debit").first()
+            if from_account.balance - fund_data['transaction_amount'] > 1000:
+                from_account.balance -= fund_data['transaction_amount']
+                db.session.add(from_account)
+                to_account.balance += fund_data['transaction_amount']
+                db.session.add(to_account)
+                transaction_status = "success"
+            else:
+                transaction_status = "fail"
```

Comment:
Also either raise exception here with some message or call ResponseGenerator with status as fail.

Reply:
Have called the response generator with fail status